src/libYARP_robotinterface/src/yarp/robotinterface/Action.cpp

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2006-2020 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#include "Action.h"
+#include "Param.h"
+
+#include <yarp/os/LogStream.h>
+
+#include <string>
+
+
+
+class yarp::robotinterface::Action::Private
+{
+public:
+    Private(Action * /*parent*/) :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'Private' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/robotology/yarp/pullRequest?prid=4763706)

Reply:
This was existing code. I agree with making it explicit, but  I prefer to do this in another PR to limit as  much as possible the changes that are not related to move the code from the tool to the library.

Reply:
However, if I prefer I can fix it.